components/Input/TextInput.tsx

```diff
@@ -24,7 +24,7 @@ export function TextInput({
   const inputMode = type === "text" ? "text" : type === "number" ? "decimal" : "email";
   // treat numbers as text inputs
   const _type = type === "number" ? "text" : type;
-  const onChange = useOnChange(onInput, maxDecimals, allowNegative);
+  const onChange = useOnChange(onInput, maxDecimals, allowNegative, _type);
```

Comment:
this was a bug i think, previously you could not edit the input box because the type would be defaulting to number

Reply:
awesome thanks for catching this